RicardoMorato/threads
# Feat/q7

## What I did
- Answered question seven;
- Added some comments about the code.

## How to test
1. Run `git fetch --all`;
2. Run `git checkout Feat/q7`;
3. Now, you have to verify if at least one image with `.ppm` extension in the directory `q7`; 
4. To compile and run the file, you have to run this code: `gcc -pthread q7.c -o exe && ./exe`
5. Run the code `ls` and see if exist the file `new-image.ppm`;
6. If everything is ok, now you can compare the images and they have to look like the image bellow.

I. With Colours (image.ppm)
![image](https://user-images.githubusercontent.com/23245711/98604968-069b4580-22bb-11eb-82de-999919f9f3c6.png)

II. Without Colours (new-image.ppm)
![image](https://user-images.githubusercontent.com/23245711/98605122-4eba6800-22bb-11eb-83c2-dde6b65b61af.png)
